test/core/templates/module-test.js

```diff
@@ -41,16 +40,9 @@ module.exports = (moduleName, options, benchmark, input) => {
       base64Img.imgSync(result, target, `${moduleName}-result`);
       base64Img.imgSync(benchmark, target, `${moduleName}-benchmark`);
 
-      result = `./test_outputs/${moduleName}-result.png`;
-      benchmark = `./test_outputs/${moduleName}-benchmark.png`;
-
-      looksSame(result, benchmark, function(err, res) {
-        if (err) console.log(err);
-
-        t.equal(res.equal, true, `${moduleName} module works correctly`);
-        sequencer = null;
-        t.end();
-      });
+      t.equal(result === benchmark, true, `${moduleName} module works correctly with png`);
```

Comment:
If we don't use looks-same anymore then please remove the dependency too.

Reply:
done

Reply:
Why is it just png here?

Reply:
Also, I am using looks-same in checking for change in option test so don't remove it!

Reply:
looks-same is giving wrong result for some reason. The example is histogram module.